crates/aggkit-prover/src/rpc.rs

```diff
@@ -46,13 +46,32 @@ impl AggchainProofGrpcService for GrpcService {
         request: Request<GenerateAggchainProofRequest>,
     ) -> Result<Response<GenerateAggchainProofResponse>, Status> {
         let request = request.into_inner();
+
+        let last_proven_block = request.last_proven_block;
+        let requested_end_block = request.requested_end_block;
+
+        info!(
+            %last_proven_block,
+            %requested_end_block,
+            l1_info_tree_root_hash=%hex::encode(
+                request
+                    .l1_info_tree_root_hash
+                    .as_ref()
+                    .map_or(&Bytes::default(), |v| &v.value)
+            ),
+            "Generate aggchain proof request"
```

Comment:
Maybe we can use the gRPC method name here to have better understanding of what's called
```suggestion
            "Receive GenerateAggchainProof request"
```